src/entities/donation.ts

```diff
@@ -126,6 +126,10 @@ export class Donation extends BaseEntity {
   @Column({ nullable: true })
   transakTransactionLink?: string;
 
+  @Field(type => Boolean, { nullable: true })
+  @Column({ nullable: true, default: false })
+  segmentNotified: boolean;
+
```

Comment:
@mohammadranjbarz what do you think? I set the default as false. But we must avoid notifying everyone once we deploy, so We would need to manually set all to true for old donations.
Then afterwards new donations catched by your script will go through my cronjob.

Reply:
I think ut's good to set it 'false` for default value, but please write a migration file to set `true` for older donations otherwise your cronjob will send notification for all older donations and it's not good at all